README.md

```diff
@@ -19,16 +19,36 @@ npm install --save mozilla-payments-client
 
 ## Development
 
-In order to develop the payments-client along side the payments-example site you can run:
+You can run a lightweight server to host the bundled payments client which is
+useful for testing as a script on another site
+(such as the [payments example site][example-site]).
+This serves a distribution bundle but also re-builds the bundle any time
+a file changes.
+
+Start the server like this:
 
 ```
 grunt serve
 ```
 
-Using [Charles Proxy](http://www.charlesproxy.com/) you can re-write to the payments-client src
-served by mozilla/payments-example with the built file containing changes you're working from.
+### Proxying with Charles
+
+The [example site][example-site] uses a bundled version of the client library by
+default so one way to try out your edits on the example site is to re-route bundle
+requests to your locally served source code bundle using
+[Charles Proxy](http://www.charlesproxy.com/).
+
+To do this import the charles rewrite config (see the Charles directory) into
+the rewrite panel.
+
+### Manually swapping in a development bundle
+
+Without Charles, you can run this command each time you change some code to keep
+the [payments example site][example-site] up to date:
 
-To do this import the charles rewrite config (see the Charles directory).
+````
+grunt build && cp dist/payments-client.js /path/to/payments-example/public/lib/js/
```

Comment:
Perhaps add a note to say that to update the example site permanently a release is required followed by an update of the package.json in the example tree.